### ISSUE TITLE
Experiments on the Truncation LineMOD

### ISSUE THREAD

**Issue by JiChun-Wang** [chensong1995/HybridPose#66]
Hi, I try to test your provided pre-trained models on the Truncation LineMOD which is introduced in the PVNet. But the evaluation result is always 0.0. I hope your explaination sincerely!
The size of the input image is 256*256 which is cropped from the LineMOD dataset, so I only change the camera intrinsic parameter in generate_data function.

**Comment by chensong1995** [comment]
Hello there,

Thanks for your interest in our work! My suggestion is to visualize the network prediction first and see whether the problem is in network regression or pose estimation. My suspension is that a constant transformation matrix is missing during the 2D-3D alignment.

I hope this helps!

**Comment by JiChun-Wang** [comment]
As your suggestion, I find the visualization results out of the network is right and a constant transformance matrix is added during the 2D-3D aligment, but the final result is still wrong, so it seems the problem is in pose estimation.

**Comment by JiChun-Wang** [comment]
And follow your ablation code to set the weights for three hybrid representations as 1, 0, 0 where the initial pose is abtained from the cv2.solvePnP, the ADD result for the object cat is 0.2714 of initial prediction but that of final prediction is only 0.0407.

**Comment by chensong1995** [comment]
Thanks for your follow-up. After you add the transformation matrix, make sure to visualize the ground truth pose labels and see if they looks right. One way to do this is to project the 3D model into 2D using the ground-truth labels and then render the model on the image. Once we are certain about the labels and alignment, you may want to set a few break points in `PoseRegression::RefinePose` and investigate the optimization trajectory. Perhaps do some calculations by hand and verify the debugger output against your hand calculation. On a new dataset, we may or may not need some hyper parameter tuning. I hope this helps!

**Comment by Y1996-wwj** [comment]
> 您好，我尝试在 PVNet 中引入的 Truncation LineMOD 上测试您提供的预训练模型。但是评估结果总是0.0。希望你真诚的解释！ 输入图像的大小为 256*256，是从 LineMOD 数据集裁剪而来的，所以我只更改了 generate_data 函数中的相机内在参数。

您好，请问Truncation_linemod数据集是您自己制作的吗？可不可以发给我一份呀？非常感谢

**Comment by Y1996-wwj** [comment]
> Hi, I try to test your provided pre-trained models on the Truncation LineMOD which is introduced in the PVNet. But the evaluation result is always 0.0. I hope your explaination sincerely! The size of the input image is 256*256 which is cropped from the LineMOD dataset, so I only change the camera intrinsic parameter in generate_data function.

您好，请问Truncation_linemod 数据集是自己制作的吗？可不可以发我一份呀？非常感谢
